bindings/python/libmata.pxd

```diff
@@ -157,16 +157,16 @@ cdef extern from "mata/nfa.hh" namespace "Mata::Nfa":
         CNfa(unsigned long, StateSet, StateSet, CAlphabet*)
 
         # Public Functions
-        void make_initial(State)
```

Comment:
asi bych vratil zpet na make, bo ona je to vlastne vlastnost stavu.

Reply:
Is it a state property? States are actually just numbers which belongs to different sets. It is implemented as well as formally defined this way.

Reply:
Martin is right that `add_` corresponds to how the library implements states. However, I feel that what we want to give the user is an interface that works as he would expect to work with the automata themselves. And there, state definitely has properties such as being final/initial.

Furthermore, the reason we have renamed the function to `make_` still stands: This function only modifies a property of the passed state (in the context of automata), it does not “add” a new state which is final/initial as well. I still believe `make_` explicitly states how the automaton changes. However, I am fine with either.

Let us discuss further, whether we want the interface to correspond to how the library implements automata or whether we want the interface to correspond to the concept of automata and operations on them in general.

Reply:
Ok this is an idea for another discussion. We should solve here how to name functions creating initial and final states.

Reply:
I am for keeping the original `make_`, for the reasons mentioned above. But I believe this was not the decision made on the last meeting.

Reply:
Ok, I reverted it to `make_` and let it be written to stone. I'm not going to change it again.